templates/docker-compose.template

```diff
@@ -48,41 +52,32 @@ services:
     restart: always
     depends_on:
       - helium-miner
-    expose:
-      - "1681"
+    network_mode: host
+    extra_hosts:
+        - "helium-miner:127.0.0.1"
     labels:
       io.balena.features.sysfs: 1
       io.balena.features.procfs: 1
     volumes:
       - nebra:/var/nebra
 
   helium-miner:
-    image: nebraltd/hm-miner:{{ARCH}}-{{MINER_VERSION}}
-    depends_on:
-      - dbus-session
-      - diagnostics
-    expose:
-      - "1680"
-      - "4467"
-    ports:
-      - "44158:44158/tcp"
```

Comment:
some people use this port for monitoring (see comments in #462 ) so we might want to leave it open

Reply:
Host mode doesn't allow port redirection or open via compose. So the port is there, but not written explicitly anymore.